### DISCUSSION TITLE
Tablet driver prevents osu! from launching

### DISCUSSION THREAD

**Discussion by C9Glax** [ppy/osu#21526] [Q&A] [answered]
### Type

Crash to desktop

### Bug description

When starting osu! with the wacom-linux-driver, the game doesn't load.
Disabling the driver `rmmod wacom` solves this, but is probably not the intended way.

### Version

2022.1123.0

### Logs

[database.log](https://github.com/ppy/osu/files/10158223/database.log)
[input.log](https://github.com/ppy/osu/files/10158224/input.log)
[network.log](https://github.com/ppy/osu/files/10158225/network.log)
[performance.log](https://github.com/ppy/osu/files/10158226/performance.log)
[runtime.log](https://github.com/ppy/osu/files/10158227/runtime.log)

**Comment by peppy** [comment, marked as answer]
No error in logs. Likely an external issue.

**Reply by C9Glax** [reply]
Probably, and there probably won't be a need for change. 👍🏼 
Wanted to document this for the future when more people run into this.
